### ISSUE TITLE
2021-07

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#12]

**Comment by qiaojy19** [comment]
# Jul - 2021


## 07.26 - 07.30
| Title                                                                                                                                                                                                                    |
| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| [A Decentralized Federated Learning Framework via Committee Mechanism with Convergence Guarantee](https://arxiv.org/abs/2108.00365)                                                                                      |
| [A Payload Optimization Method for Federated Recommender Systems](https://arxiv.org/abs/2107.13078)                                                                                                                      |
| [A Scalable Federated Multi-agent Architecture for Networked Connected Communication Network](https://arxiv.org/abs/2108.00506)                                                                                          |
| [Accelerated Gradient Descent Learning over Multiple Access Fading Channels](https://arxiv.org/abs/2107.12452)                                                                                                           |
| [Aggregate or Not? Exploring Where to Privatize in DNN Based Federated Learning Under Different Non-IID Scenes](https://arxiv.org/abs/2107.11954)                                                                        |
| [D-Cliques: Compensating NonIIDness in Decentralized Federated Learning with Topology](https://arxiv.org/abs/2104.07365)                                                                                                 |
| [Decentralized Deep Learning for Mobile Edge Computing: A Survey on Communication Efficiency and Trustworthiness](https://arxiv.org/abs/2108.03980)                                                                      |
| [Distributed Learning for Time-varying Networks: A Scalable Design](https://arxiv.org/abs/2108.00231)                                                                                                                    |
| [Fed-EINI: An Efficient and Interpretable Inference Framework for Decision Tree Ensembles in Federated Learning](https://arxiv.org/abs/2105.09540)                                                                       |
| [Federated Learning Meets Natural Language Processing: A Survey](https://arxiv.org/abs/2107.12603)                                                                                                                       |
| [Federated Learning with Unreliable Clients: Performance Analysis and Mechanism Design](https://arxiv.org/abs/2105.06256)                                                                                                |
| [Fedlearn-Algo: A flexible open-source privacy-preserving machine learning platform](https://arxiv.org/abs/2107.04129)                                                                                                   |
| [HAFLO: GPU-Based Acceleration for Federated Logistic Regression](https://arxiv.org/abs/2107.13797)                                                                                                                      |
| [Learning-based Prediction, Rendering and Transmission for Interactive Virtual Reality in RIS-Assisted Terahertz Networks](https://arxiv.org/abs/2107.12943)                                                             |
| [LEGATO: A LayerwisE Gradient AggregaTiOn Algorithm for Mitigating Byzantine Attacks in Federated Learning](https://arxiv.org/abs/2107.12490)                                                                            |
| [Lithography Hotspot Detection via Heterogeneous Federated Learning with Local Adaptation](https://arxiv.org/abs/2107.04367)                                                                                             |
| [New Metrics to Evaluate the Performance and Fairness of Personalized Federated Learning](https://arxiv.org/abs/2107.13173)                                                                                              |
| [On The Impact of Client Sampling on Federated Learning Convergence](https://arxiv.org/abs/2107.12211)                                                                                                                   |
| [Packet Routing with Graph Attention Multi-agent Reinforcement Learning](https://arxiv.org/abs/2107.13181)                                                                                                               |
| [Preliminary Steps Towards Federated Sentiment Classification](https://arxiv.org/abs/2107.11956)                                                                                                                         |
| [Private Retrieval, Computing and Learning: Recent Progress and Future Challenges](https://arxiv.org/abs/2108.00026)                                                                                                     |
| [QuPeD: Quantized Personalization via Distillation with Applications to Federated Learning](https://arxiv.org/abs/2107.13892)                                                                                            |
| [Secure Bayesian Federated Analytics for Privacy-Preserving Trend Detection](https://arxiv.org/abs/2107.13640)                                                                                                           |
| [Sensing and Mapping for Better Roads: Initial Plan for Using Federated Learning and Implementing a Digital Twin to Identify the Road Conditions in a Developing Country -- Sri Lanka](https://arxiv.org/abs/2107.14551) |
| [Stronger Privacy for Federated Collaborative Filtering with Implicit Feedback](https://arxiv.org/abs/2105.03941)                                                                                                        |
| [Towards Industrial Private AI: A two-tier framework for data and model security](https://arxiv.org/abs/2107.12806)                                                                                                      |

## 07.19 - 07.23
| Title                                                                                                                                                |
| ---------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Accelerating Federated Edge Learning via Optimized Probabilistic Device Scheduling](https://arxiv.org/abs/2107.11588)                               |
| [CANITA: Faster Rates for Distributed Convex Optimization with Communication Compression](https://arxiv.org/abs/2107.09461)                          |
| [Coded Privacy-Preserving Computation at Edge Networks](https://arxiv.org/abs/2106.08290)                                                            |
| [Communication Efficiency in Federated Learning: Achievements and Challenges](https://arxiv.org/abs/2107.10996)                                      |
| [Decentralized federated learning of deep neural networks on non-iid data](https://arxiv.org/abs/2107.08517)                                         |
| [Defending against Reconstruction Attack in Vertical Federated Learning](https://arxiv.org/abs/2107.09898)                                           |
| [Device Scheduling and Update Aggregation Policies for Asynchronous Federated Learning](https://arxiv.org/abs/2107.11415)                            |
| [Fed-ensemble: Improving Generalization through Model Ensembling in Federated Learning](https://arxiv.org/abs/2107.10663)                            |
| [Federated Causal Inference in Heterogeneous Observational Data](https://arxiv.org/abs/2107.11732)                                                   |
| [Federated Graph Classification over Non-IID Graphs](https://arxiv.org/abs/2106.13423)                                                               |
| [Federated Learning using Smart Contracts on Blockchains, based on Reward Driven Approach](https://arxiv.org/abs/2107.10243)                         |
| [Federated Learning Versus Classical Machine Learning: A Convergence Comparison](https://arxiv.org/abs/2107.10976)                                   |
| [Federated Learning with Buffered Asynchronous Aggregation](https://arxiv.org/abs/2106.06639)                                                        |
| [Federated Learning with Fair Worker Selection: A Multi-Round Submodular Maximization Approach](https://arxiv.org/abs/2107.11728)                    |
| [FedHealth 2: Weighted Federated Transfer Learning via Batch Normalization for Personalized Healthcare](https://arxiv.org/abs/2106.01009)            |
| [FedProto: Federated Prototype Learning over Heterogeneous Devices](https://arxiv.org/abs/2105.00243)                                                |
| [How Does Cell-Free Massive MIMO Support Multiple Federated Learning Groups?](https://arxiv.org/abs/2107.09577)                                      |
| [Local SGD Optimizes Overparameterized Neural Networks in Polynomial Time](https://arxiv.org/abs/2107.10868)                                         |
| [Personalized Federated Learning over non-IID Data for Indoor Localization](https://arxiv.org/abs/2107.04189)                                        |
| [Precision-Weighted Federated Learning](https://arxiv.org/abs/2107.09627)                                                                            |
| [Relay-Assisted Cooperative Federated Learning](https://arxiv.org/abs/2107.09518)                                                                    |
| [Renyi Differential Privacy of the Subsampled Shuffle Model in Distributed Learning](https://arxiv.org/abs/2107.08763)                               |
| [Responsible and Regulatory Conform Machine Learning for Medicine: A Survey of Technical Challenges and Solutions](https://arxiv.org/abs/2107.09546) |
| [RingFed: Reducing Communication Costs in Federated Learning on Non-IID Data](https://arxiv.org/abs/2107.08873)                                      |
| [RoFL: Attestable Robustness for Secure Federated Learning](https://arxiv.org/abs/2107.03311)                                                        |
| [Simultaneous Wireless Information and Power Transfer for Federated Learning](https://arxiv.org/abs/2104.12749)                                      |
| [Subgraph Federated Learning with Missing Neighbor Generation](https://arxiv.org/abs/2106.13430)                                                     |
| [Trends in Blockchain and Federated Learning for Data Sharing in Distributed Platforms](https://arxiv.org/abs/2107.08624)                            |

## 07.12 - 07.16
| Title                                                                                                                                                          |
| -------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Field Guide to Federated Optimization](https://arxiv.org/abs/2107.06917)                                                                                    |
| [An Experimental Study of Data Heterogeneity in Federated Learning Methods for Medical Imaging](https://arxiv.org/abs/2107.08371)                              |
| [AutoFL: Enabling Heterogeneity-Aware Energy Efficient Federated Learning](https://arxiv.org/abs/2107.08147)                                                   |
| [Business analytics meets artificial intelligence: Assessing the demand effects of discounts on Swiss train tickets](https://arxiv.org/abs/2105.01426)         |
| [Communication-Efficient Hierarchical Federated Learning for IoT Heterogeneous Systems with Imbalanced Data](https://arxiv.org/abs/2107.06548)                 |
| [Concept drift detection and adaptation for federated and continual learning](https://arxiv.org/abs/2105.13309)                                                |
| [Decentralized and Personalized Federated Learning](https://arxiv.org/abs/2107.07190)                                                                          |
| [Decentralized Bayesian Learning with Metropolis-Adjusted Hamiltonian Monte Carlo](https://arxiv.org/abs/2107.07211)                                           |
| [DeFed: A Principled Decentralized and Privacy-Preserving Federated Learning Algorithm](https://arxiv.org/abs/2107.07171)                                      |
| [Federated Action Recognition on Heterogeneous Embedded Devices](https://arxiv.org/abs/2107.12147)                                                             |
| [Federated Mixture of Experts](https://arxiv.org/abs/2107.06724)                                                                                               |
| [Federated Self-Training for Semi-Supervised Audio Recognition](https://arxiv.org/abs/2107.06877)                                                              |
| [Federated Whole Prostate Segmentation in MRI with Personalized Neural Architectures](https://arxiv.org/abs/2107.08111)                                        |
| [Genetic CFL: Optimization of Hyper-Parameters in Clustered Federated Learning](https://arxiv.org/abs/2107.07233)                                              |
| [IFedAvg: Interpretable Data-Interoperability for Federated Learning](https://arxiv.org/abs/2107.06580)                                                        |
| [OmniLytics: A Blockchain-based Secure Data Market for Decentralized Machine Learning](https://arxiv.org/abs/2107.05252)                                       |
| [One-Point Gradient-Free Methods for Composite Optimization with Applications to Distributed Optimization](https://arxiv.org/abs/2107.05951)                   |
| [Personalized Federated Learning via Maximizing Correlation with Sparse and Hierarchical Extensions](https://arxiv.org/abs/2107.05330)                         |
| [Privacy-preserving Spatiotemporal Scenario Generation of Renewable Energies: A Federated Deep Generative Learning Approach](https://arxiv.org/abs/2107.07738) |
| [RobustFed: A Truth Inference Approach for Robust Federated Learning](https://arxiv.org/abs/2107.08402)                                                        |
| [Separate but Together: Unsupervised Federated Learning for Speech Enhancement from Non-IID Data](https://arxiv.org/abs/2105.04727)                            |
| [TEACHING -- Trustworthy autonomous cyber-physical applications through human-centred intelligence](https://arxiv.org/abs/2107.06543)                          |
| [Towards Practical Watermark for Deep Neural Networks in Federated Learning](https://arxiv.org/abs/2105.03167)                                                 |

## 07.05 - 07.09
| Title                                                                                                                                                               |
| ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Federated Semi-Supervised Learning Approach for Network Traffic Classification](https://arxiv.org/abs/2107.03933)                                                |
| [DER Forecast using Privacy Preserving Federated Learning](https://arxiv.org/abs/2107.03248)                                                                        |
| [Differentially private federated deep learning for multi-site medical image segmentation](https://arxiv.org/abs/2107.02586)                                        |
| [End-to-End Speech Recognition from Federated Acoustic Models](https://arxiv.org/abs/2104.14297)                                                                    |
| [Energy Efficient Federated Learning in Integrated Fog-Cloud Computing Enabled Internet-of-Things Networks](https://arxiv.org/abs/2107.03520)                       |
| [FedAdapt: Adaptive Offloading for IoT Devices in Federated Learning](https://arxiv.org/abs/2107.04271)                                                             |
| [Federated Learning as a Mean-Field Game](https://arxiv.org/abs/2107.03770)                                                                                         |
| [Federated Learning for Multi-Center Imaging Diagnostics: A Study in Cardiovascular Disease](https://arxiv.org/abs/2107.03901)                                      |
| [Federated Learning with Downlink Device Selection](https://arxiv.org/abs/2107.03510)                                                                               |
| [Federated Learning with Dynamic Transformer for Text to Speech](https://arxiv.org/abs/2107.08795)                                                                  |
| [Management of Resource at the Network Edge for Federated Learning](https://arxiv.org/abs/2107.03428)                                                               |
| [Memory-aware curriculum federated learning for breast cancer classification](https://arxiv.org/abs/2107.02504)                                                     |
| [Optimizing the Numbers of Queries and Replies in Federated Learning with Differential Privacy](https://arxiv.org/abs/2107.01895)                                   |
| [SplitAVG: A heterogeneity-aware federated deep learning method for medical imaging](https://arxiv.org/abs/2107.02375)                                              |
| [Towards Node Liability in Federated Learning: Computational Cost and Network Overhead](https://arxiv.org/abs/2107.02006)                                           |
| [UAV-assisted Online Machine Learning over Multi-Tiered Networks: A Hierarchical Nested Personalized Federated Learning Approach](https://arxiv.org/abs/2106.15734) |
